client/packages/common/src/ui/layout/tables/material-react-table/useGetColumnDefDefaults.tsx

```diff
@@ -81,6 +84,22 @@ export const useGetColumnTypeDefaults = () => {
           },
         };
 
+      case ColumnType.NumberInput:
```

Comment:
Just get rid of this one and do inline, like you have for the other edit columns?